ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_histograms.py

```diff
@@ -45,3 +123,53 @@ def test_numpy_bincount(
         weights=None,
         minlength=0,
     )
+
+
+@handle_frontend_test(fn_tree="numpy.histogram_bin_edges", data=_histogram_helper())
+def test_numpy_histogram_bin_edges(
+    data,
+    on_device,
+    fn_tree,
+    frontend,
+    backend_fw,
+    test_flags,
+):
+    input_dtype, a, bins, range, weights, density = data
+    print(input_dtype)
+    helpers.test_frontend_function(
+        input_dtypes=[input_dtype],
+        frontend=frontend,
+        backend_to_test=backend_fw,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        a=a,
+        bins=bins,
+        range=range,
+        weights=weights,
+    )
+
+
+@handle_frontend_test(fn_tree="numpy.histogram", data=_histogram_helper())
+def test_numpy_histogram(
```

Comment:
Showing some function related errors: `TypeError: histogram() got multiple values for argument 'density'`